### DISCUSSION TITLE
Windows安装的Navidrome局域网无法访问吗？

### DISCUSSION THREAD

**Discussion by sucooer** [navidrome/navidrome#3235] [Q&A]
我在Windows11安装了Navidrome,并用nssm创建了服务，本机访问没有问题，但是局域网其它设备却无法连接，是win系统的原因还是Navidrome的原因？

**Comment by deluan** [comment]
Probably because some configuration in Windows or its firewall, as we have lots of other users using it in Windows without any issues. Unfortunately I don't use Windows so I can't help with that. Hopefully someone will jump in to try to help you.

PS: Please post in English.

**Reply by omw123a** [reply]
局域网防火墙端口未开放，需要开放对应端口，可以看这个https://www.bilibili.com/opus/1048095113988800512
